### PR TITLE
Refactor after merging individual branches

### PR DESCRIPTION
**Pull request by Jake0Miller**

### CODE REVIEW

**Comment by ktsune** [comment]
It's all coming together!

File: lib/game.rb [hunk 1]
```diff
@@ -1,9 +1,12 @@
 class Game
-  attr_reader :game_row, :home_team_id, :away_team_id, :away_goals, :home_goals, :outcome, :season
+  attr_reader :game_row, :game_id, :home_team_id, :away_team_id,
+              :away_goals, :home_goals, :outcome, :season
```
Comment (djflx) [inline lib/game.rb:3]: was the game_id just missing here?

File: modules/league_statistics.rb [hunk 2]
```diff
@@ -98,7 +98,7 @@ def lowest_scoring_home_team
   end
 
   def total_wins(home = true, away = true)
-    wins = @games.inject({}) do |hash, game|
+    @games.inject({}) do |hash, game|
```
Comment (djflx) [inline modules/league_statistics.rb:101]: i love/hate inject, please tell me more about this part!

File: modules/team_information.rb [hunk 3]
```diff
@@ -23,28 +19,26 @@ def games_by_season(team_id)
     end
   end
 
-  def home_won?(game)
-    game.home_goals > game.away_goals
+  def won_at_home?(team_id,game)
+    game.home_team_id == team_id && game.home_goals > game.away_goals
```
Comment (djflx) [inline modules/team_information.rb:23]: you're just adding an extra bit of logic here ya?

**Review by djflx — APPROVED** [review]
This looks great, although I'm still a little confused as to why this runs but our version yesterday didn't?? I know it probably has to do with our dummy_data.

**Pull request merged**